numba/targets/arraymath.py

```diff
@@ -1415,6 +1414,21 @@ def check_dimensions(array_like, name):
             if isinstance(array_like.key[0].key[0], types.Sequence):
                 raise TypeError("{0} has more than 2 dimensions".format(name))
 
+@register_jitable
+def _handle_ddof(ddof):
+    if not np.isfinite(ddof):
+        raise TypeError('Cannot convert non-finite ddof to integer')
```

Comment:
Should this be a `ValueError`? The function is accepting a floating point type, it's just that it must be finite?

Reply:
Yeah, I made this a ValueError initially and can't quite recall why I changed it - so it's back to a ValueError